store/mongo/migration_1_2_13.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2022 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package mongo
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/mendersoftware/deployments/model"
+)
+
+type migration_1_2_13 struct {
+	client *mongo.Client
+	db     string
+}
+
+type OldImage struct {
+	// Image ID
+	Id string `json:"id" bson:"_id" valid:"uuidv4,required"`
+
+	// User provided field set
+	*model.ImageMeta `bson:"meta"`
+
+	// Field set provided with yocto image
+	*OldArtifactMeta `bson:"meta_artifact"`
+
+	// Artifact total size
+	Size int64 `json:"size" bson:"size" valid:"-"`
+
+	// Last modification time, including image upload time
+	Modified *time.Time `json:"modified" valid:"-"`
+}
+
+// Information provided by the Mender Artifact header
+type OldArtifactMeta struct {
+	// artifact_name from artifact file
+	Name string `json:"name" bson:"name" valid:"length(1|4096),required"`
+
+	// Compatible device types for the application
+	//nolint:lll
+	DeviceTypesCompatible []string `json:"device_types_compatible" bson:"device_types_compatible" valid:"length(1|4096),required"`
+
+	// Artifact version info
+	Info *model.ArtifactInfo `json:"info"`
+
+	// Flag that indicates if artifact is signed or not
+	Signed bool `json:"signed" bson:"signed"`
+
+	// List of updates
+	Updates []model.Update `json:"updates" valid:"-"`
+
+	// Provides is a map of artifact_provides used
+	// for checking artifact (version 3) dependencies.
+	//nolint:lll
+	Provides map[string]string `json:"artifact_provides,omitempty" bson:"provides,omitempty" valid:"-"`
+
+	// Depends is a map[string]interface{} (JSON) of artifact_depends used
+	// for checking/validate against artifact (version 3) provides.
+	Depends map[string]interface{} `json:"artifact_depends,omitempty" bson:"depends" valid:"-"`
+
+	// ClearsProvides is a list of strings (JSON) of clears_artifact_provides used
+	// for clearing already-installed artifact (version 3) provides.
+	//nolint:lll
+	ClearsProvides []string `json:"clears_artifact_provides,omitempty" bson:"clears_provides,omitempty" valid:"-"`
+}
```

Comment:
nit: Only use the `Id` and `Provides` fields are used in this struct, the rest can be stripped away.